src/app/containers/Ad/Amp/index.jsx

```diff
@@ -103,7 +103,7 @@ const AMP_ACCESS_DATA = endpoint => ({
 const LABEL_LINK = 'https://www.bbc.com/usingthebbc/cookies/';
 
 export const AMP_ACCESS_FETCH = service => {
-  const togglesEndpoint = `${process.env.SIMORGH_TOGGLES_URL}/toggles?application=simorgh&service=${service}`;
+  const togglesEndpoint = `${process.env.SIMORGH_CONFIG_URL}?application=simorgh&service=${service}`;
```

Comment:
```suggestion
  const configEndpoint = `${process.env.SIMORGH_CONFIG_URL}?application=simorgh&service=${service}`;
```

Reply:
I think these consts can still remain as `togglesEndpoint` because in this context what we are getting is the toggles